readme.md

```diff
@@ -132,6 +132,23 @@ slugify('foo@unicorn', {
 //=> 'foo-at-unicorn'
 ```
 
+##### customReplacements
+
+Type: `boolean`\
+Default: `false`
+
+Allow leading underscore as an escape `_foo_bar` → `_foo-bar`.
```

Comment:
It's not really an "escape". I think it would be more correct to say that it **preserves** leading underscore.

Reply:
hmm understood gonna change it today!